crates/build/re_types_builder/src/objects.rs

```diff
@@ -464,6 +464,22 @@ impl Object {
             })
             .collect();
 
+        if is_enum {
+            assert!(
+                !fields.is_empty(),
+                "enums must have at least one variant, but {fqname} has none",
+            );
+
+            assert!(
+            fields[0].name == "Invalid" && fields[0].enum_value == Some(0),
+            "enums must start with 'Invalid' variant with value 0, but {fqname} starts with {} = {:?}",
+            fields[0].name,
+            fields[0].enum_value,
+        );
+
+            fields.remove(0); // Remove the 'Invalid' variant.
```

Comment:
It would be good with a comment here explaining why we require this only to then remove it